src/ca65/main.c

```diff
@@ -962,6 +969,7 @@ int main (int argc, char* argv [])
         { "--verbose",             0,      OptVerbose              },
         { "--version",             0,      OptVersion              },
         { "--warnings-as-errors",  0,      OptWarningsAsErrors     },
+        { "--segment-list",        0,      OptSeglist              },
```

Comment:
The new line should go after "--relax-checks" (options are alphabetically sorted)